lib/ansible/modules/cloud/cloudscale/cloudscale_server.py

```diff
@@ -203,12 +203,12 @@
 from time import sleep
 
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.six.moves.urllib.parse import urlencode
 from ansible.module_utils.urls import fetch_url
 
 
 API_URL = 'https://api.cloudscale.ch/v1/'
-TIMEOUT_WAIT = 30
+TIMEOUT_WAIT = 60
+TIMEOUT_API = 30
```

Comment:
Because it probably depends on the user's network, would it help to let the user overwrite the timeout by a param e.g. `api_timeout`, so these values are only used as default?